code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -350,6 +350,15 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 	var/relative_threat = LORENTZ_DISTRIBUTION(threat_curve_centre, threat_curve_width)
 	threat_level = round(lorentz_to_amount(relative_threat), 0.1)
 
+	var/most_hours_played = 0
+	for (var/mob/plr in GLOB.player_list)
+		var/mob/living/carbon/human/hm = plr
+		if (istype(hm))
+			var/datum/job/J = SSjob.GetJob(hm.job)
+			if (J && J.department_flag == ENGSEC)
```

Comment:
why is engineering involved

Reply:
engineering and security share department flag

Reply:
so an engineer with high hours is fucking over a sec team with low hours?